app/views/model_view.py

```diff
@@ -38,12 +37,12 @@ def post(self, request):
                 want_receipt="wantReceipt" in request.POST,
                 telephone_number=request.POST["telephoneNumber"],
                 mobile_number=request.POST["mobileNumber"],
-                address_line=request.POST["addressLine"],
+                address_line_one=request.POST["addressLineOne"],
```

Comment:
I believe these views model_views are deprecated and not in use anymore as it was for a manual donor input form. 

https://github.com/CodeTheChangeUBC/reBOOT/blob/3aff13c3ba6cf9fadee48ce2bd0c7366c304f2e3/app/views/views.py#L29

Reply:
I'd suggest testing/updating other views such as api_view or data_view

Reply:
Ah, that's good to know! Are you aware of any other deprecated code? It would be helpful to eliminate it to clarify test coverage.

Reply:
I added new test cases in https://github.com/CodeTheChangeUBC/reBOOT/pull/338/commits/2674773b76b85d6fd5a00ea801cf64545f3985cf and https://github.com/CodeTheChangeUBC/reBOOT/pull/338/commits/a0f9391820dacb8492efa3b82efe16e0e321494a.